tensorboard/webapp/plugins/plugins_component.ts

```diff
@@ -79,7 +90,6 @@ export class PluginsComponent implements OnChanges {
 
     const pluginElement = this.createPlugin(plugin);
     if (pluginElement) {
-      pluginElement.id = plugin.id;
```

Comment:
Now that components are created via `createComponent`, Angular started to complain due to this line, because it didn't expect the id to be modified.  These `id`s were only used for testing purposes, so I updated the tests.